rotkehlchen/data/importer.py

```diff
@@ -586,7 +593,11 @@ def _import_cryptocom_associated_entries(
                     base_amount_bought = deserialize_asset_amount(
                         credited_row['Amount'],
                     ) * part_of_total
-                    rate = Price(abs(quote_amount_sold / base_amount_bought))
+
+                    if quote_amount_sold != ZERO:
+                        rate = Price(abs(quote_amount_sold / base_amount_bought))
```

Comment:
Hm ... isn't the problem here if `base_amount_bought` is zero? Aren't you trying to protect for division by zero?

Reply:
Here the case where `base_amount_bought` is when `quote_amount_sold` is too but I agree that is better to protect also against division by zero by changing it